benchmarks/v4.1.0.js

```diff
@@ -0,0 +1,688 @@
+"use strict";
```

Comment:
Since this is copypasta dist code from the current version, I don't think this should be checked in